src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolSscsJudicialOfficeMappingTest.java

```diff
@@ -120,7 +123,9 @@ void shouldReturnSalariedRoles(String appointment, String appointmentType,
         "Tribunal Member Financially Qualified,Fee Paid,BBA3,'fee-paid-financial,hmcts-judiciary','372'",
         "Tribunal Member Financially Qualified,Fee Paid,BBA3,'fee-paid-financial,hmcts-judiciary','362'",
         "Member of the First-tier Tribunal,Fee Paid,BBA3,'fee-paid-financial,hmcts-judiciary','372'",
-        "Member of the First-tier Tribunal,Fee Paid,BBA3,'fee-paid-financial,hmcts-judiciary','362'"
+        "Member of the First-tier Tribunal,Fee Paid,BBA3,'fee-paid-financial,hmcts-judiciary','362'",
+        "Member of the First-tier Tribunal (sitting in retirement),Fee Paid,BBA3,"
+                + "'fee-paid-financial,hmcts-judiciary','362'"
```

Comment:
Should this not be `fee-paid-disability`.

I think this test only verifies roleAssignments match in length not in content.  Can you update?  i.e. in for each check rolename is in expected list.

Can yo uverify other test sin same class do the same.

Reply:
good spot. updated test with `fee-paid-disability` and RA content check. See commit [3386555](https://github.com/hmcts/am-org-role-mapping-service/pull/1883/commits/3386555b66ddc8dc80e5a62817b92f7b6bb4b6cd).

All other tests in class that take in `expectedRoles` param are fine as they verify RA content.